plotsScript.py

```diff
@@ -2427,5 +2429,5 @@ def makeRachelPlots(depth=depth, fontSize=18):
     # makeRachelPlots(depth='udeep')
     # xdColExtFitScoresComb()
 
-    # xdColExtFitScores()
-    makeRachelPlots(depth='udeep')
+    # xdColExtFitScores(trainClfs=True)
```

Comment:
This file relies on commenting in/out calls to individual functions in order to modify what gets run. In this case, I was modifying how that function should be run (if were selected to be run).